src/Components/ErrorBoundary.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class ErrorBoundary extends Component {
```

Comment:
You also need to define `getDerivedStateFromError` method

Reply:
fixed